docs/data/advanced-components/overview.md

```diff
@@ -0,0 +1,172 @@
+
+If the license key is missing, the component will look something like this:
+
+<div class="only-light-mode">
+  <img src="/static/x/watermark-light.png" style="width: 653px; margin-bottom: 2rem;" alt="" loading="lazy">
+</div>
+<div class="only-dark-mode">
+  <img src="/static/x/watermark-dark.png" style="width: 645px; margin-bottom: 2rem;" alt="" loading="lazy">
+</div>
+
+> Note that you are still allowed to use the component for [evaluation purposes](#evaluation-trial-licenses) in this case.
+
+#### License key expired
+
+The licenses are perpetual, the license key will work forever with the current version of the software.
+
+However, **access to updates/upgrades** is not perpetual.
+Installing a version of the component released after the license key has expired will trigger a watermark and console message.
+For example, if you purchase a one-year license today, you are not licensed to install a version released two years in the future, but you can optionally update to any version, including major versions, if it's released in the next 12 months.
+
+#### Invalid license key
+
+The license key you have installed is not as issued by MUI.
+
+## Support
+
+### GitHub
+
+We use GitHub issues as a bug and feature request tracker. If you think you have found a bug, or have a new feature idea, please start by making sure it hasn't already been [reported or fixed](https://github.com/mui/mui-x/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed). You can search through existing issues and pull requests to see if someone has reported one similar to yours.
+
+[Open an issue](https://github.com/mui/mui-x/issues/new/choose)
+
+### StackOverflow
+
+For crowdsourced answers from expert MUI developers in our community.
+StackOverflow is also visited from time to time by the maintainers of MUI.
+
+[Post a question](https://stackoverflow.com/questions/tagged/mui)
+
+### Professional support
+
+When purchasing an MUI X Pro or Premium license you get access to professional support for a limited duration.
+Support is available on multiple channels, but the recommended channel is GitHub issues.
+If you need to share private information, you can also use email.
+
+- **MUI X Pro**: No SLA is provided but MUI's maintainers give these issues more attention than the ones from the Community plan. The channels:
+  - GitHub: [Open a new issue](https://github.com/mui/mui-x/issues/new/choose) and leave your Order ID.
+  - Email (only to share private information): [Open a new issue](https://support.mui.com/hc/en-us/requests/new?tf_360023797420=mui_x) or send an email at x@mui.com.
+- **MUI X Premium**: Same as MUI X Pro, but with priority over Pro, and a 48 hour SLA for the first answer.
+  - GitHub: this plan is not available yet
+  - Emails: this plan is not available yet
+- **MUI X Premium Priority**: Same as MUI X Premium but with a 24 hours SLA for the first answer.
+  - GitHub: this plan is not available yet
+  - Emails: this plan is not available yet
```

Comment:
Is this necessary since these plans are not yet available?

Reply:
Copy / Pasted, so no strong feeling here
@oliviertassinari